tests/browser/add-event-listener-options.browser.js

```diff
@@ -4,10 +4,8 @@
  */
 
 const jil = require('jil')
-let matcher = require('../../tools/jil/util/browser-matcher')
```

Comment:
You do not need to remove these lines from the test for it to work, as they are not evaluated anyway.  But these could be removed for clarity in all browser test files if desired.